elements/indium.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: page
+title: Sulfur
```

Comment:
Ooops! Forgot to update the page title. Can you change this to indium?

Reply:
Hi, I just updated the title.

Reply:
👍🏻